tests/feature/test_description.py

```diff
@@ -1,22 +1,48 @@
 """Test descriptions."""
-from pytest_bdd import scenario
 
+import textwrap
 
-def test_description(request):
+from pytest_bdd import given, scenario
+
+
+@given("I have a bar")
+def bar():
+    return "bar"
+
+
+def test_description(request, testdir):
     """Test description for the feature."""
+    feature = testdir.makefile(
+        ".feature",
+        description=textwrap.dedent(
+            """
+        Feature: Description
 
-    @scenario("description.feature", "Description")
-    def test():
-        pass
+            In order to achieve something
+            I want something
+            Because it will be cool
 
-    assert (
-        test.__scenario__.feature.description
-        == """In order to achieve something
-I want something
-Because it will be cool
 
+            Some description goes here.
 
-Some description goes here."""
+            Scenario: Description
+                Given I have a bar
+        """
+        ),
     )
 
-    test(request)
+    @scenario(feature.strpath, "Description")
+    def test_description(request):
+        pass
+
+    test_description(request)
```

Comment:
we should use testdir.runpytest instead too. `test_description` could check `test_description.__scenario__.feature.description`